plutus-playground-server/src/Playground/Interpreter.hs

```diff
@@ -214,6 +214,7 @@ runghcOpts =
     , "-fobject-code"
     -- FIXME: stupid GHC bug still
     , "-package plutus-tx"
+    -- , "-package plutus-wallet-api"
```

Comment:
Committed in error?

Reply:
no I don't think so, if you want to run the tests in stack you _might_ need to swap these round so I just left it there to make that easier.